en/editor/preferences/index.md

```diff
@@ -76,6 +76,14 @@ The **Asset database** tab is used to set the [Assets](../assets/index.md) panel
 - **Ignore (Regular)**: use a regular expression and fill in the path to a specific asset file, then that asset will be ignored.
 - **Default Meta**: used to set the default configuration when importing assets within a project. Please refer to the description below for details.
 
+### Update resources automatically
+
+When we do resource manipulation outside of Creator, back in Creator. Creator does a resource check. When the number of files is too high, or the disk random read and write speed is too low, it will cause the resource system response lag. For example, when you click on a resource, the Inspector takes a while to show up.
```

Comment:
```suggestion
When performing resource manipulation outside of Creator and bringing the assets into Creator, Creator does a resource check. When the number of files is too high, or the disk random read and write speed is too low, it will cause the resource system response lag. For example, when clicking on a resource, the Inspector takes a while to show up.
```